src/mlagility/cli/cli.py

```diff
@@ -259,10 +276,24 @@ def main():
     # )
 
     #######################################
-    # Parser for the "report" command
+    # Subparser for the "cache" command
     #######################################
 
-    report_parser = subparsers.add_parser(
+    cache_parser = subparsers.add_parser(
+        "cache", help="Commands for managing the build cache"
+    )
+
+    cache_subparsers = cache_parser.add_subparsers(
+        title="cache",
+        help="Commands for managing the build cache",
+        required=True,
+    )
```

Comment:
simply running "benchit cache" crashes beautifully. Can we fix that so people actually get a useful message (similar to what is shown when only "benchit" is executed without any args?

Reply:
Good catch, thanks! Fixed it. It turns out that `benchit benchmark` is even worse though... filed #66 to deal with that one later.